package.json

```diff
@@ -58,6 +58,7 @@
     "poolee": "1.0.1",
     "request": "2.74.0",
     "scrypt-hash": "1.1.13",
+    "sha1": "1.1.1",
```

Comment:
The documentation at [1] suggests this is a pure-js implementation of sha1.  Is that intentional?  It seems like it could be unnecessarily slow.  An alternative would be to either use the primitives from the built-in `crypto` module or, if we're going to pull in an external package anyway, to use a faster hash like siphash [2] that will directly give us an integer.

[1] https://www.npmjs.com/package/sha1
[2] https://www.npmjs.com/package/siphash